src/Service/Ecr/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Added
 
 - AWS api-change: Added `eu-central-2` region
+- AWS api-change: This release includes a fix in the DescribeImageScanFindings paginated output.
```

Comment:
None of these change log entries are relevant, and we're missing a note for the new eu region.

Reply:
actually, I used GitHub come t suggestion to fix the generated comments.

But I'm at SymfonyCon without my laptop, and github's mobile version don't let me comment several lines 😅 

I'll have a look this week-end

Reply:
lol. have a good conf. ;)